sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -2165,9 +2168,10 @@ export interface CpkInfo {
 }
 
 // @public
-export abstract class Credential implements RequestPolicyFactory {
+abstract class Credential_2 implements RequestPolicyFactory {
     create(_nextPolicy: RequestPolicy, _options: RequestPolicyOptions): RequestPolicy;
 }
+export { Credential_2 as Credential }
```

Comment:
Mainly I'm fine with the changes. Just curious, do you know why api-extractor makes this change?

Reply:
I believe it's because Credential shadows an existing global type - I think this one? https://github.com/microsoft/TypeScript/blob/main/lib/lib.dom.d.ts#L3635

Please see https://github.com/Azure/azure-sdk-for-js/pull/17917#issuecomment-930465924 and https://github.com/Azure/azure-sdk-for-js/pull/17917#issue-1010274710 for reference. TL;DR - API Extractor is _exposing_ any time we shadow a global type.

From the consumer's perspective there are no changes - but this is the main reason we held off upgrading for so long